frappe/installer.py

```diff
@@ -124,6 +140,52 @@ def install_db(root_login=None, root_password=None, db_name=None, source_sql=Non
 	frappe.flags.in_install_db = False
 
 
+def find_org(org_repo: str) -> Tuple[str, str]:
+	from frappe.exceptions import InvalidRemoteException
+	import requests
+
+	org_repo = org_repo[0]
```

Comment:
```suggestion
```

This function is specific to its use in `fetch_details_from_tag`. We should maybe move this bit in the caller function